resources/web/wwi/image_loader.js

```diff
@@ -83,7 +84,15 @@ function _loadImage(src) {
     img.onload = () => {
       resolve(img);
     };
-    img.onerror = () => console.error('Error in loading: ' + src);
+    img.onerror = () => {
+      if (typeof img.failed === 'undefined') {
+        console.error('Error in loading: ' + src);
+        img.src = 'https://cyberbotics.com/wwi/images/missing_texture.pn';
+        img.failed = true;
+      } else
+        // Fail to get the missing texture image, exit to avoid an infinite loop
```

Comment:
This should never happen unless the `missing_texture.png` file is missing, right?
If so, we should probably print an error message in the console here.

Reply:
It could happen if there is a internet cut-off right at the wrong moment. 
Highly unlikely but not impossible.

A message is automatically printed in the console when WebotsJS fail to download an URL